client/app/achievements/achievements.less

```diff
@@ -1,3 +1,7 @@
 ng-invald {
   color: red
 }
+
+.event-description > img {
+  width: 100%;
```

Comment:
Would the markdown styler ever apply extra styling to pictures? If not, then I think it is fair to add a max-width: 100% to all pictures. On the other hand, the width: 100% would certainly blow away any styling and should maybe be switched to max-width as well